### PR TITLE
LIMS-6675 : In case the unit field with sub & super, allow this to di…

### PR DESCRIPTION
**Pull request by NouranYoussefMahasseb**
```
D:\1lims-automation> nosetests -vs -m test066 --logging-level=WARNING ui_testing/testcases/basic_tests/test06_orders.py --tc-file=config.ini
c:\users\lenovo\appdata\local\programs\python\python38-32\lib\site-packages\nose\plugins\manager.py:394: RuntimeWarning: Unable to load plugin noseprogressive = noseprogressive:ProgressivePlugin: No module named '
_curses'
  warn("Unable to load plugin %s: %s" % (ep, e),
2020-08-12 20:14:20.748 | INFO     | api_testing.apis.base_api:_get_authorized_session:46 - Get authorized api session.
2020-08-12 20:14:20.752 | INFO     | api_testing.apis.base_api:_get_authorized_session:47 - admin:admin
2020-08-12 20:14:21.700 | INFO     | api_testing.apis.base_api:_get_authorized_session:58 - session ID : FlXWS-PvgWVLwf-bWAsgH6nvLgjoqI4wVUKySfqDfUU .....

DevTools listening on ws://127.0.0.1:55372/devtools/browser/7a088425-95ca-464e-851e-5e6d9f9ae829
New: Test unit: Export: In case the unit field with sub & super, [with type='Quantitative'] ...
2020-08-12 20:15:34.893 | INFO     | ui_testing.testcases.base_test:setUp:27 - Test case : test066_test_unit_with_sub_and_super_unit_name_appears_in_unit_field_drop_down_0_Quantitative
2020-08-12 20:15:41.036 | INFO     | ui_testing.pages.base_pages:wait_until_page_is_loaded:562 - wait until page is loaded
2020-08-12 20:15:41.084 | DEBUG    | ui_testing.pages.base_pages:sleep_tiny:39 - wait up to 0.5 sec
2020-08-12 20:15:42.012 | INFO     | api_testing.apis.orders_api:set_configuration:395 - set order configuration
2020-08-12 20:15:43.436 | INFO     | api_testing.apis.base_api:set_configuration:73 - status code: 1
2020-08-12 20:15:43.444 | INFO     | ui_testing.pages.testunits_page:get_test_units_page:11 -  + Get test units page.
2020-08-12 20:15:48.317 | INFO     | ui_testing.pages.base_pages:wait_until_page_is_loaded:562 - wait until page is loaded
2020-08-12 20:15:48.374 | DEBUG    | ui_testing.pages.base_pages:sleep_tiny:39 - wait up to 0.5 sec
2020-08-12 20:15:49.503 | INFO     | ui_testing.pages.testunits_page:open_configurations:94 - open testunits configurations
2020-08-12 20:15:49.975 | DEBUG    | ui_testing.pages.base_pages:sleep_tiny:39 - wait up to 0.5 sec
2020-08-12 20:15:51.680 | DEBUG    | ui_testing.pages.base_pages:sleep_small:43 - wait up to 1 sec
2020-08-12 20:15:55.458 | INFO     | ui_testing.pages.testunits_page:open_testunit_name_configurations_options:102 - open testunits name configurations options
2020-08-12 20:15:55.482 | DEBUG    | ui_testing.pages.base_pages:sleep_tiny:39 - wait up to 0.5 sec
2020-08-12 20:15:57.431 | DEBUG    | ui_testing.pages.base_pages:sleep_tiny:39 - wait up to 0.5 sec
2020-08-12 20:15:59.136 | DEBUG    | ui_testing.pages.base_pages:sleep_tiny:39 - wait up to 0.5 sec
2020-08-12 20:16:00.303 | INFO     | ui_testing.pages.testunits_page:clear_all_selected_view_and_search_options:111 - clear selected view and search options of testunits name
2020-08-12 20:16:00.499 | DEBUG    | ui_testing.pages.base_pages:sleep_tiny:39 - wait up to 0.5 sec
2020-08-12 20:16:09.565 | DEBUG    | ui_testing.pages.base_pages:sleep_small:43 - wait up to 1 sec
2020-08-12 20:16:12.268 | DEBUG    | ui_testing.pages.base_pages:sleep_small:43 - wait up to 1 sec
2020-08-12 20:16:18.607 | INFO     | ui_testing.testcases.basic_tests.test06_orders:test066_test_unit_with_sub_and_super_unit_name_appears_in_unit_field_drop_down:2319 - Create new Quantitative testunit
2020-08-12 20:16:18.615 | INFO     | api_testing.apis.base_api:wrapper:122 - GET : https://automation.1lims.com/api/testUnits
2020-08-12 20:16:19.746 | INFO     | api_testing.apis.base_api:wrapper:129 - Status code: 1
2020-08-12 20:16:19.754 | INFO     | ui_testing.testcases.basic_tests.test06_orders:test066_test_unit_with_sub_and_super_unit_name_appears_in_unit_field_drop_down:2329 - go to orders page
2020-08-12 20:16:23.525 | INFO     | ui_testing.pages.base_pages:wait_until_page_is_loaded:562 - wait until page is loaded
2020-08-12 20:16:25.069 | DEBUG    | ui_testing.pages.base_pages:sleep_tiny:39 - wait up to 0.5 sec
2020-08-12 20:16:26.171 | INFO     | ui_testing.pages.order_page:create_new_order_get_test_unit_suggetion_list:687 -  Create new order.
2020-08-12 20:16:26.175 | INFO     | ui_testing.pages.orders_page:click_create_order_button:21 - Press create order button
2020-08-12 20:16:26.911 | INFO     | ui_testing.pages.base_pages:wait_until_page_is_loaded:562 - wait until page is loaded
2020-08-12 20:16:27.720 | DEBUG    | ui_testing.pages.base_pages:sleep_tiny:39 - wait up to 0.5 sec
2020-08-12 20:16:29.411 | DEBUG    | ui_testing.pages.base_pages:sleep_small:43 - wait up to 1 sec
2020-08-12 20:16:31.881 | INFO     | ui_testing.pages.order_page:set_new_order:18 - Set new order.
2020-08-12 20:16:45.932 | DEBUG    | ui_testing.pages.base_pages:sleep_tiny:39 - wait up to 0.5 sec
2020-08-12 20:16:55.384 | DEBUG    | ui_testing.pages.base_pages:sleep_tiny:39 - wait up to 0.5 sec
2020-08-12 20:16:57.803 | DEBUG    | ui_testing.pages.base_pages:sleep_small:43 - wait up to 1 sec
2020-08-12 20:17:05.786 | DEBUG    | ui_testing.pages.base_pages:sleep_tiny:39 - wait up to 0.5 sec
2020-08-12 20:17:07.710 | DEBUG    | ui_testing.pages.base_pages:sleep_small:43 - wait up to 1 sec
2020-08-12 20:17:10.211 | INFO     | ui_testing.pages.order_page:create_new_order_get_test_unit_suggetion_list:697 - get test unit suggestion list
2020-08-12 20:17:16.257 | INFO     | ui_testing.testcases.base_test:tearDown:33 - go to dashboard page
2020-08-12 20:17:18.841 | INFO     | ui_testing.testcases.base_test:tearDown:35 - TearDown.     
ok

----------------------------------------------------------------------
Ran 1 test in 178.634s

OK


```

### CODE REVIEW

File: ui_testing/pages/order_page.py [hunk 1]
```diff
@@ -679,4 +679,23 @@ def create_new_order_get_test_unit_suggetion_list(self, material_type, test_unit
         self.set_article(article='')
         self.sleep_small()
         self.info('get test unit suggestion list')
-        test_units = self.base_selenium.get_drop_down_suggestion_list(element='order:test_unit',item_text=test_unit_name)
\ No newline at end of file
+        test_units = self.base_selenium.get_drop_down_suggestion_list(element='order:test_unit',item_text=test_unit_name)
+
+
+
```
Comment (ghada-37) [inline ui_testing/pages/order_page.py:685]: remove this Duplicated function (lime 671 to 685)

File: ui_testing/testcases/basic_tests/test06_orders.py [hunk 2]
```diff
@@ -22,6 +25,8 @@ class OrdersTestCases(BaseTest):
     def setUp(self):
         super().setUp()
         self.order_page = Order()
+        self.test_units_page = TstUnits()
+        self.test_unit_page=TstUnit()
```
Comment (ghada-37) [inline ui_testing/testcases/basic_tests/test06_orders.py:29]: move these lines to your test case

File: ui_testing/pages/order_page.py (hunk 1 above)
Comment (NouranYoussefMahasseb) [reply ui_testing/pages/order_page.py:671]: done

File: ui_testing/testcases/basic_tests/test06_orders.py (hunk 2 above)
Comment (NouranYoussefMahasseb) [reply ui_testing/testcases/basic_tests/test06_orders.py]: done

**Pull request closed without merging**